vnext/ReactUWP/Views/ViewViewManager.cpp

```diff
@@ -49,6 +49,19 @@ class ViewShadowNode : public ShadowNodeBase {
                 "topAccessibilityTap",
                 std::move(folly::dynamic::object("target", m_tag)));
         });
+
+    DynamicAutomationProperties::SetAccessibilityActionEventHandler(
+        panel, [=](winrt::react::uwp::AccessibilityAction const &action) {
+          folly::dynamic eventData = folly::dynamic::object("target", m_tag);
+
+          if (action.Label.empty()) {
+            eventData.insert("actionName", HstringToDynamic(action.Name));
+          } else {
+            eventData.insert("actionName", HstringToDynamic(action.Label));
```

Comment:
>actionName [](start = 30, length = 10)

should all these strings be externalized?

Reply:
I made change to `eventData.insert(
              "actionName", action.Label.empty() ? HstringToDynamic(action.Name) : HstringToDynamic(action.Label));` and will not make it `constexp`